prototype/distribution/features/northbound-ogc-sensorthings-mqtt-feature/pom.xml

```diff
@@ -0,0 +1,239 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <!--
+  /*********************************************************************
+  * Copyright (c) 2023 Kentyou and others
+  *
+  * This program and the accompanying materials are made
+  * available under the terms of the Eclipse Public License 2.0
+  * which is available at https://www.eclipse.org/legal/epl-2.0/
+  *
+  * SPDX-License-Identifier: EPL-2.0
+  **********************************************************************/
+  -->
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.sensinact.gateway.distribution.features</groupId>
+    <artifactId>features</artifactId>
+    <version>0.0.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>northbound-ogc-sensorthings-mqtt-feature</artifactId>
+  <name>Eclipse sensiNact Northbound OGC Sensorthings MQTT Feature</name>
+  <description>This project contains a feature providing an OGC Sensorthings compatible MQTT extension for sensiNact</description>
+  <packaging>pom</packaging>
+
+  <properties>
+    <sensinact.version>0.0.2-SNAPSHOT</sensinact.version>
+    <sensorthings.version>0.0.2-SNAPSHOT</sensorthings.version>
+    <moquette.version>0.16</moquette.version>
+    <netty.version>4.1.77.Final</netty.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.moquette</groupId>
+      <artifactId>moquette-broker</artifactId>
+      <version>${moquette.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.sensinact.gateway.northbound.sensorthings</groupId>
+      <artifactId>mqtt</artifactId>
+      <version>${sensorthings.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.sensinact.gateway.northbound.sensorthings</groupId>
+      <artifactId>dto</artifactId>
+      <version>${sensorthings.version}</version>
+      <scope>test</scope>
```

Comment:
Doesn't this need to be `compile` scope to make sure it's included in the repo zip?

```suggestion
```